__tests__/clients/email/email.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { createEmailClient } from '@clients/email/email.client';
+
+describe('Creating email client ', () => {
```

Comment:
What do you think about testing the functionality itself besides the initialization of the client?

Reply:
Not Apply